### PR TITLE
Change the default property used to identify which elements to ignore

### PR DESCRIPTION
**Pull request by roesslerj**

### CODE REVIEW

File: docs/recheck/usage/filter.md [hunk 1]
```diff
@@ -142,8 +142,8 @@ matcher: $key=$value
 
 | `$key`     | `$value` (Example)                   | Description                                                                            |
 | ---------- | ------------------------------------ | -------------------------------------------------------------------------------------- |
-| `xpath`    | `html[1]/body[1]/div[1]`             | Note that this does only supports absolute XPaths.<br>*This is the default mechanism.* |
-| `retestid` | `div-b4f23`                          | A unique, stable ID for an element, that is generated by ***recheck***.                |
+| `retestid` | `div-b4f23`                          | A unique, stable ID for an element, that is generated by ***recheck***.<br>*This is the default mechanism.* |
```
Comment (diba1013) [inline docs/recheck/usage/filter.md]: Please format the table accordingly

Comment (roesslerj) [reply docs/recheck/usage/filter.md]: What do you mean accordingly? 
Screen Shot 2019-11-13 at 13 11 19  21 34 28

Comment (roesslerj) [reply docs/recheck/usage/filter.md]: This is what is looks like in Eclipse. What do you want it to look like? If you have a specific indentation in mind, please go and format it like you want...

Comment (diba1013) [reply docs/recheck/usage/filter.md]: If you use a proper editor 😉, the table will be automatically formatted, such that all cells of a single column are the same width (i.e. the pipe `|` are directly beneath each other). I will do the formatting.

**Comment by diba1013** [comment]
To be merged with the 1.7.0 release.

**Pull request merged**